client/src/components/CreatePost/CreatePost.js

```diff
@@ -209,8 +112,6 @@ const CreatePost = ({ onCancel, loadPosts, ...props }) => {
       value={{ form, setForm, currentStep, setCurrentStep, postId, setPostId }}
     >
       <Wrapper onCancel={clearState} {...props}>
-        <Step1 gtmPrefix={props.gtmPrefix} />
-        <Step2 user={props.user} gtmPrefix={props.gtmPrefix} />
         <Step4 gtmPrefix={props.gtmPrefix} />
       </Wrapper>
       <Step3 onCancel={clearState} {...props} gtmPrefix={props.gtmPrefix} />
```

Comment:
These can be renamed to step 1 and 2, to not cause confusion in the future.

Reply:
👍  will do